TECHNICAL_MANIFEST.md

```
@@ -0,0 +1,1618 @@
+
+**Step 3: Evaluate with DeepEval** (if CSV workflow needed)
+```bash
+python deepeval_eva.py  # Reads answers.csv
+```
+
+### Latest Evaluation Results
+
+**Recent Scores (from `evaluation/results/`):**
+
+**DeepEval (December 2, 2024):**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)